src/options.c

```diff
@@ -81,6 +83,44 @@ static int parse_ipv4(struct super_block *sb, char *str,
 	return 0;
 }
 
+static int parse_metadev_path(struct super_block *sb, substring_t *substr,
```

Comment:
This doesn't enforce that it's a metadev, that's the caller's notion.  Maybe call it parse_bdev_path()?